package-lock.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Please commit changes to `yarn.lock`, don't make a `package-lock.json`.

Reply:
Fixed